packages/ng-excel-builder/integration/tests/helpers/file-suites/test-4-options.xls

```diff
@@ -1,131 +1,56 @@
-                    <Data ss:Type="String">locale.lang</Data>
-                </Cell>
-            </Row>
-            <Row ss:Height="40">
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">1</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">herschel</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">32</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">uk</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">en</Data>
-                </Cell>
-            </Row>
-            <Row ss:Height="40">
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">2</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">hermann</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">42</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">us</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">en</Data>
-                </Cell>
-            </Row>
-            <Row ss:Height="40">
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">3</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">-</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">19</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">at</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">de</Data>
-                </Cell>
-            </Row>
-            <Row ss:Height="40">
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">4</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">hawking</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="Number">42</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">ru</Data>
-                </Cell>
-                <Cell ss:StyleID="BodyCellStyle">
-                    <Data ss:Type="String">ru</Data>
-                </Cell>
-            </Row>
-        </Table>
-    </Worksheet>
-</Workbook>
+﻿
```

Comment:
you can prettify test xmls for better review, test suites minify all files so they pass successfully